src/client/eppo-client.ts

```diff
@@ -979,12 +979,12 @@ export default class EppoClient {
    * Enqueues an arbitrary event. Events must have a type and a payload.
    * TODO: enforce max message size
    */
-  track(type: string, event: unknown) {
+  track(type: string, payload: Record<string, unknown>) {
```

Comment:
strictly speaking this is a breaking change since we're narrowing the public API here, so it will require a major bump. This method is not really used anywhere yet, though, so I'd be OK bumping minor version instead, if people feel strongly against bumping major